src/helm/meet/values.yaml

```diff
@@ -305,3 +305,184 @@ posthog:
+  args: []
+
+  ## @param celery.replicas Amount of celery replicas
+  replicas: 1
+
+  ## @param celery.shareProcessNamespace Enable share process namespace between containers
+  shareProcessNamespace: false
+
+  ## @param celery.sidecars Add sidecars containers to celery deployment
+  sidecars: []
+
+  ## @param celery.migrateJobAnnotations Annotations for the migrate job
+  migrateJobAnnotations: {}
+
+  ## @param celery.securityContext Configure celery Pod security context
+  securityContext: null
+
+  ## @param celery.envVars Configure celery container environment variables
+  ## @extra celery.envVars.BY_VALUE Example environment variable by setting value directly
+  ## @extra celery.envVars.FROM_CONFIGMAP.configMapKeyRef.name Name of a ConfigMap when configuring env vars from a ConfigMap
+  ## @extra celery.envVars.FROM_CONFIGMAP.configMapKeyRef.key Key within a ConfigMap when configuring env vars from a ConfigMap
+  ## @extra celery.envVars.FROM_SECRET.secretKeyRef.name Name of a Secret when configuring env vars from a Secret
+  ## @extra celery.envVars.FROM_SECRET.secretKeyRef.key Key within a Secret when configuring env vars from a Secret
+  ## @skip celery.envVars
+  envVars:
+    <<: *commonEnvVars
+
+  ## @param celery.podAnnotations Annotations to add to the celery Pod
+  podAnnotations: {}
+
+  ## @param celery.service.type celery Service type
+  ## @param celery.service.port celery Service listening port
+  ## @param celery.service.targetPort celery container listening port
+  ## @param celery.service.annotations Annotations to add to the celery Service
+  service:
+    type: ClusterIP
+    port: 80
+    targetPort: 8000
+    annotations: {}
+
+  ## @param celery.probes.liveness.path [nullable] Configure path for celery HTTP liveness probe
+  ## @param celery.probes.liveness.targetPort [nullable] Configure port for celery HTTP liveness probe
+  ## @param celery.probes.liveness.initialDelaySeconds [nullable] Configure initial delay for celery liveness probe
+  ## @param celery.probes.liveness.initialDelaySeconds [nullable] Configure timeout for celery liveness probe
+  ## @param celery.probes.startup.path [nullable] Configure path for celery HTTP startup probe
+  ## @param celery.probes.startup.targetPort [nullable] Configure port for celery HTTP startup probe
+  ## @param celery.probes.startup.initialDelaySeconds [nullable] Configure initial delay for celery startup probe
+  ## @param celery.probes.startup.initialDelaySeconds [nullable] Configure timeout for celery startup probe
+  ## @param celery.probes.readiness.path [nullable] Configure path for celery HTTP readiness probe
+  ## @param celery.probes.readiness.targetPort [nullable] Configure port for celery HTTP readiness probe
+  ## @param celery.probes.readiness.initialDelaySeconds [nullable] Configure initial delay for celery readiness probe
+  ## @param celery.probes.readiness.initialDelaySeconds [nullable] Configure timeout for celery readiness probe
+  probes: {}
```

Comment:
@qbey any recommendation on this one?

Reply:
Nope, but maybe @rouja can reuse configuration from other project?